dcc-portal-pql/src/test/java/org/dcc/portal/pql/es/visitor/special/EntitySetVisitorTest.java

```diff
@@ -178,6 +178,15 @@ public void fileRepoIdentifiableTermTest() {
     val termNode = getTermNode(result);
 
     assertThat(termNode.getField()).isEqualTo("id");
+    assertLookupInfo(termNode.getLookup(), "file-ids");
+  }
+
+  @Test
+  public void fileRepoDonorIdentifiableTermTest() {
+    val result = prepareResult(getQuery("donor"), FILE);
+    val termNode = getTermNode(result);
+
+    assertThat(termNode.getField()).isEqualTo("donors.donor_id");
     assertLookupInfo(termNode.getLookup(), "donor-ids");
```

Comment:
Can we centralize [this](https://github.com/icgc-dcc/dcc-portal/search?utf8=%E2%9C%93&q=%22donor-ids%22) at least within the context of `dcc-portal-pql`?